lib/faker/string.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Faker.String do
+  @moduledoc """
+  Function for generating Strings
+  """
+
+  @doc """
+  Returns a ramdom String
+  """
+  @spec random_string(integer) :: String.t
+  def random_string(length \\ 8) do
```

Comment:
I'd suggest renaming it to `base64`. Why?

- The `random` prefix is not needed: Everything is random in Faker
- It doesn't return any string, it returns base64.

`Faker.String.base64` feels quite clear to me